paf-mvp-core-js/src/express/node.ts

```diff
@@ -80,4 +96,27 @@ export class Node implements INode {
       this.logger.Error(endPointName, err);
     };
   }
+
+  /**
+   * Build an handler that validate the body of the Request
+   * against a JSON schema.
+   * @returns the built hander
+   */
+  protected buildJsonBodyValidatorHandler(jsonSchema: string): RequestHandler {
+    return (req: Request, res: Response, next: NextFunction) => {
+      const validation = this.jsonValidator.validate(jsonSchema, req.body as string);
+      if (!validation.isValid) {
+        const details = validation.errors.map((e) => e.message).join(' - ');
+        const error: ClientNodeError = {
+          type: ClientNodeErrorType.INVALID_JSON_BODY,
+          details: details,
```

Comment:
[NIT] you can simply write `details` instead of `details: details`

Reply:
Yes, this is a matter of taste I guess but I don't hold it very strongly this style. Updated.